stats/stats_central.php

```diff
@@ -6,11 +6,22 @@
 include_once($relPath.'ThemedTable.inc');
 include_once($relPath.'site_news.inc');
 include_once($relPath.'misc.inc');
+include_once('./cumulative_total_proj_summary_graph.inc');
 
 require_login();
 
 $title = _("Statistics Central");
-output_header($title);
+
+$cumulative_total_proj_summary_graph_data = cumulative_total_proj_summary_graph();
+output_header($title, SHOW_STATSBAR, [
+    "js_files" => [
```

Comment:
Let's create a `get_graph_js_files()` function in the now-common PHP include file that returns these two JS files. That way we have a central place we reference the D3 files and return various chart-type JS files. This means we need to update the `pinc/3rdpart/d3/README.md` on the filename to update if it changes.